coderd/workspaces_test.go

```diff
@@ -3441,7 +3441,7 @@ func TestWorkspaceUsageTracking(t *testing.T) {
 	})
 }
 
-func TestNotifications(t *testing.T) {
+func TestWorkspaceNotifications(t *testing.T) {
```

Comment:
Redundant change because this is already in a file named "workspaces".

Reply:
I did that because the package is `api` so I can't use `TestNotifications` term again in the `templates_test.go` for example.

Reply:
Updated the name to `TestNotifyDeletedTemplate` following the users_test.go example.